arch/esp32/esp32c3.ini

```diff
@@ -4,3 +4,7 @@ custom_esp32_kind = esp32c3
 
 monitor_speed = 115200
 monitor_filters = esp32_c3_exception_decoder
+
+build_flags =
+  ${esp32_base.build_flags}
+  -D ARCH_ESP32_C3
```

Comment:
please don't do that. there are predefined macros to find out which flavour of ESP32 you are on.